geode-core/src/distributedTest/java/org/apache/geode/cache/query/dunit/DefaultQueryServiceDeprecationTest.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package org.apache.geode.cache.query.dunit;
+
+import static org.apache.geode.distributed.internal.DistributionConfig.GEMFIRE_PREFIX;
+
+import java.io.File;
+import java.io.Serializable;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+
+import org.apache.geode.cache.query.internal.DefaultQueryService;
+import org.apache.geode.test.assertj.LogFileAssert;
+import org.apache.geode.test.dunit.rules.ClusterStartupRule;
+import org.apache.geode.test.dunit.rules.MemberVM;
+
+public class DefaultQueryServiceDeprecationTest implements Serializable {
+  MemberVM locator;
+
+  @Rule
+  public ClusterStartupRule cluster = new ClusterStartupRule();
+
+  @Before
+  public void startUp() {
+    locator = cluster.startLocatorVM(0);
+  }
+
+  @After
+  public void shutDown() {
+    cluster.stop(0);
+  }
```

Comment:
I don't think this is needed, the `ClusterStartupRule` already stops every started member and restores the system properties after each test method. See [here](https://github.com/apache/geode/blob/develop/geode-dunit/src/main/java/org/apache/geode/test/dunit/rules/ClusterStartupRule.java#L159-L186) for more details.

Reply:
Done